crates/sparrow-compiler/src/ast_to_dfg.rs

```diff
@@ -242,20 +243,7 @@ pub fn add_to_dfg(
                 // recreate them?
                 let behavior = function.tick_behavior().context("tick behavior")?;
                 if let Ok(agg_input) = dfg.get_binding("$condition_input") {
-                    // The argument is a tick, so we can directly create the necessary node.
-                    let agg_input_op = dfg.operation(agg_input.value());
-                    let tick_input = smallvec![agg_input_op];
-                    let tick_node = dfg.add_operation(Operation::Tick(behavior), tick_input)?;
-                    let tick_node = Arc::new(AstDfg::new(
-                        tick_node,
-                        tick_node,
-                        FenlType::Concrete(DataType::Boolean),
-                        agg_input.grouping(),
-                        agg_input.time_domain().clone(),
-                        function_name.location().clone(),
-                        None,
-                    ));
-                    return Ok(tick_node);
+                    return add_tick(dfg, behavior, agg_input.value());
```

Comment:
Ah okay -- so this is still valid for coming from fenl, doing the weird condition input binding, but through python we pass the input directly. Nice.